practice_2/WeatherApp/VIew/Cell/ForecastCell.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+class ForecastCell: UITableViewCell {
+    static let identifier = "forecastCell"
+
+    @IBOutlet private weak var dateLabel: UILabel!
+    @IBOutlet private weak var temperatureLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    func configure(by forecast: Forecast, unit: String) {
+        if let unixtime = forecast.unixtime, let temperature = forecast.temperature {
+            dateLabel.text = convertUnixtimeToDate(unixtime: unixtime)
+            temperatureLabel.text = "\(String(temperature)) \(unit)"
+        }
+    }
+
+    private func convertUnixtimeToDate (unixtime: Int) -> String {
+        let date = NSDate(timeIntervalSince1970: TimeInterval(unixtime))
+        let formatter = DateFormatter()
```

Comment:
creating DateFormatter every time is quite heavy and resourceful
 create it once and just reuse if possible

Reply:
Thanks for that point.
Made formatter as static let.